yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
where is this coming from?

Reply:
https://classic.yarnpkg.com/lang/en/docs/yarn-lock/#toc-check-into-source-control

Reply:
Hum... okay they are already in fe/.

Not.sure why it ended at the root for me. Ill remove